conda/environments/cuml_dev_cuda11.0.yml

```diff
@@ -14,6 +14,8 @@ dependencies:
 - libcumlprims=21.08.*
 - dask-cudf=21.08.*
 - dask-cuda=21.08.*
+- dask>=2021.6.0
+- distributed>=2021.6.0
```

Comment:
Why change from installing from main? Since CI is installing from main, we should keep this so dev environments behave like CI (and mirror cudf https://github.com/rapidsai/cudf/blob/0a4e8a1c860518c9e30f5289dea07bd0879fb642/conda/environments/cudf_dev_cuda11.2.yml#L64)

Reply:
That's still done above. Peter is just setting a minimum version

Reply:
No, I think what Dante was asking for is that the conda dev environment files install Dask/Distributed main to match the same as CI is running on. I did that in https://github.com/rapidsai/cuml/pull/3978/commits/80c340d12a60ceb0f54fef2195c57126d9133aac now.